### PR TITLE
add error for zero censoring weights

### PR DESCRIPTION
**Pull request by CrystalXuR**

### CODE REVIEW

**Comment by CrystalXuR** [comment]
@erikcs Could you help me take a look at this error? 
Screen Shot 2022-04-15 at 10 01 45 AM

File: r-package/survlearners/R/surv_fl_grf.R [hunk 1]
```diff
@@ -69,6 +69,9 @@ surv_fl_grf <- function(X, Y, W, D, t0, W.hat = NULL, cen.fit = "Kaplan-Meier",
     cen.times.index <- findInterval(U, c.fit$failure.times)
     C.hat <- C.hat[cbind(1:length(U), cen.times.index)]
   }
+  if (any(C.hat == 0)) {
+    stop("Some or all uncensored probabilities are exactly zeros. Check input variables or consider adjust the time of interest t0.")
```
Comment (erikcs) [inline r-package/survlearners/R/surv_fl_grf.R:73]: I don't understand what "check input variables" is supposed to mean

Comment (CrystalXuR) [reply r-package/survlearners/R/surv_fl_grf.R:73]: The input variable Y may be so large due to using the inconsistent units, e.g., Y is in days but t0 is in years

Comment (CrystalXuR) [reply r-package/survlearners/R/surv_fl_grf.R:73]: Say when the units of the input variables are inconsistent:  Y is in years but t0 is in days

**Pull request merged**